src/helpers/helperFunctions.ts

```diff
@@ -177,3 +180,30 @@ export function generatePasscodes(count: number): string[] {
 
 	return passcodes;
 }
+
+export async function getWinnerlist(examId: string): Promise<string[]> {
```

Comment:
Bu helper function degil exam.service icinde olmali, we null getirebilmeli donusunde cunku getById de kullaniyosun. Tavsiyem  extended exam document donmesi icin ayri bir endpoint ve service metodu kullanman. getExamDetails tarzinda. Bu sayede get-started kismina gereksiz veri gondermeyiz